crates/node/src/network_server/service.rs

```diff
@@ -44,6 +45,9 @@ impl NetworkServer {
         // -- HTTP service (for prometheus et al.)
         let axum_router = axum::Router::new()
             .route("/metrics", get(render_metrics))
+            .route("/debug/pprof/heap", get(pprof::heap))
```

Comment:
perhaps?
```suggestion
            .route("/debug/pprof/dump_heap", get(pprof::heap))
```

Reply:
/heap is pretty common as its what go uses by default, so would probably keep this one